device/bgp/evpn/bgpevpnethernetsegment.yaml

```diff
@@ -0,0 +1,64 @@
+components:
+  schemas:
+    Bgp.EvpnEthernetSegment:
+      type: object
+      properties:
+        esi:
+          description: >-
+            10-octet Ethernet segment identifier (esi)
+            Example - for example for multi-home nonZero ESI '10000000000000000000' .
+          type: string          
+          format: hex
+          default : '00000000000000000000'
+        active_mode:
```

Comment:
active mode can be bool as well as we have only 2 selection,can be enable_multi_home as well

Reply:
I think it would be more readable if we keep the modes as an enum because user would be able to set the mode properly as per the requirement.